package.json

```diff
@@ -87,6 +87,10 @@
     "tslint-config-prettier": "^1.15.0",
     "tslint-plugin-prettier": "^2.0.1"
   },
+  "resolutions": {
```

Comment:
Why do we need these resolutions? When can we remove them (maybe make an issue to track that)?

Reply:
TypeScript doesn’t allow us to cast `any` to type `unique symbol` until more recently, and its the only way we can polyfill Symbol and type it correctly. We should likely update to the latest TS when possible (brocolli-typescript-transpiler needs to update). There are some other breaking changes on latest though

Reply:
I think https://github.com/glimmerjs/glimmer-vm/issues/946 kind of tracks this

Reply:
Opened #965